modules/core/include/opencv2/core/va_intel_interop.hpp

```diff
@@ -0,0 +1,36 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#ifndef OPENCV_CORE_VA_INTEL_INTEROP_HPP
+#define OPENCV_CORE_VA_INTEL_INTEROP_HPP
```

Comment:
1. Do we really need this as OpenCV public API? Do users need this?
2. Non-public headers should go into "detail" subdirectory.

Reply:
Alright

Reply:
Done b56bc32